apps/cyberstorm-storybook/stories/components/Switch.stories.tsx

```diff
@@ -22,10 +22,41 @@ const SameStateSwitchTemplate: StoryFn<typeof Switch> = () => {
     </div>
   );
 };
+const SwitchLabelIdTemplate: StoryFn<typeof Switch> = () => {
+  const [state, setState] = useState(false);
+  return (
+    <div>
+      <label htmlFor="my-switch">My switch</label>
+      <div>
+        <Switch state={state} onChange={setState} labelId="my-switch" />
+      </div>
+    </div>
+  );
+};
+const SwitchLabelWithoutNestingTemplate: StoryFn<typeof Switch> = () => {
```

Comment:
Why is this named "SwitchLabel**Without**NestingTemplate" when it indeed does use nesting?

Reply:
Just brainfart while thinking describing name. Will change name to have opposite meaning 👍